rsts/community/troubleshoot.rst

```diff
@@ -14,21 +14,23 @@ Here are a couple of techniques we believe would help you jump out of the pandor
 Troubles with ``flytectl sandbox start``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- The process hangs at ``Waiting for Flyte to become ready...`` for a while
-- OR ends with a message ``Timed out while waiting for the datacatalog rollout to be created``
+- The process hangs at ``Waiting for Flyte to become ready...`` for a while; OR
+- The process ends with a message ``Timed out while waiting for the datacatalog rollout to be created``
+- To fix this:
```

Comment:
Can you move this block to "debug yourself"? Also, can you add bullet points to the "debug yourself" section? There's only one now.